data_scripts/parseTermsToElastic.js

```diff
@@ -8,7 +8,7 @@ var client = new elasticsearch.Client({
     log: 'info'
 })
 
-var db = level('/data/genenetwork/level/new/dbexternal_uint16be', {valueEncoding: 'binary'})
```

Comment:
this would break the current GN, better to perhaps make the location configurable, or to rename the metabrainnetwork folder to genenetwork when wanting to deploy with that version